cli/generators/crud/crudGenerator.js

```diff
@@ -0,0 +1,23 @@
+var Generator = require('yeoman-generator');
+var ejs = require('ejs');
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+  }
+
+  appendTpl(templatePath, destinationPath, data) {
+    var fs = this.fs;
+    ejs.renderFile(templatePath, data, (err, renderedTemplate) => {
+      if(err) throw err;
+
+      fs.copy(
+        destinationPath,
```

Comment:
Yes it's correct. Since the read/write file methods are deprecated, the proposed approach is to use the same path in from, to arguments in order to replace the existing file.